### DISCUSSION TITLE
Meshing generation to add a ground on a vehicle CFD analysis

### DISCUSSION THREAD

**Discussion by cggailla** [mdolab/MACH-Aero#168] [Q&A]
Hello, 

I am actually doing a project to add a ground on a previous vehicle analysis. 

The previous project work well but symbolize a free stream. By adding a ground, we want to inspect the effect on aero. That was conducted using pyHyp to generate a mesh around the shape of the vehicle. 

Is you solution capable to do so? and do you have some idea or tools that I can use? I thought about cutting the actual mesh at a certain height (right below the vehicle) or using the overset meshing by adding a block below the vehicle. 

Would I be able to specify appropriate boundary condition on this ground surface ?

Thanks for your help !

**Comment by hntter** [comment]
hi cggailla, I met same question as yours to simulate a building with ground. I tried to set BC boundary as xyConst in pyHyp but the mesh is still strange. Have you solved your problem?
